src/app/client/src/app/modules/player-helper/components/course-info/course-info.component.ts

```diff
@@ -9,7 +10,19 @@ export class CourseInfoComponent {
 
   @Input() courseHierarchy;
   showContentCreditsModal = false;
+  showCredits = false;
 
   constructor(public resourceService: ResourceService) { }
 
+  ngOnChanges() {
+    this.checkContentCreditAvailability();
+  }
+
+  checkContentCreditAvailability() {
+    /* istanbul ignore else */
+    if (this.courseHierarchy.copyright || this.courseHierarchy.creators || this.courseHierarchy.attributions ||
```

Comment:
use object destructuring like
`const {copyright, creators, attributions, originData, contentType} = this.courseHierarchy`